api/src/websocket/websocket.gateway.spec.ts

```
@@ -64,22 +72,106 @@ describe('WebsocketGateway', () => {
     });
-    ioClient.connect();
-    ioClient.disconnect();
+
+    socket1.disconnect();
+  });
+
+  describe('joinNotificationSockets', () => {
+    it('should make socket1 and socket3 join the room MESSAGE', async () => {
+      const [socket1, , socket3] = sockets;
+
+      [socket1, , socket3].forEach((socket) => {
```

Biome (1.9.4)

[error] This array contains an empty slot.

Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)